.github/workflows/tests.yml

```diff
@@ -298,17 +298,3 @@ jobs:
           viewerUrl: https://canonical.tiobe.com/tiobeweb/TICS/api/cfg?name=GoProjects
           ticsAuthToken: ${{ secrets.TICSAUTHTOKEN }}
           installTics: true
-
-      # This type of the TICS run compares current state of the code against the last
-      # persistent measurement point. We run in on proposed pull requests.
-      - name: "Comparative TICS run (mode: client)"
-        if: ${{ github.event_name == 'pull_request' }}
```

Comment:
Could we make it run when there is access to the token instead of removing it?

As @azkaratea pointed out OOB, the MIR project appears to have an example of this:
https://github.com/canonical/mir/blob/863945876865f272ae6be3c0ccf4eaaa0d64ea29/.github/workflows/tics.yml#L20-L21